src/unstable-temp/DownloadImageModal/FormModel.tsx

```diff
@@ -208,9 +208,7 @@ const FormControl = ({ onModelChange, options, model }: FormControlProps) => {
 								name={options.name}
 								value={
 									!!model[options.name]
-										? moment(model[options.name] as string)
-												.utc()
-												.format()
+										? parseISO(model[options.name] as string).toISOString()
```

Comment:
would the date returned in `model[options.name]` will always be an ISO string conforming to ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)? If it does then using `new Date()` or `Date.parse()` should also work.  We wouldn't even need `date-fns`